lib/util.js

```diff
@@ -793,14 +793,16 @@ var util = {
   promisifyMethod: function promisifyMethod(methodName, PromiseDependency) {
     return function promise() {
       var self = this;
+      var args = Array.prototype.slice.call(arguments);
```

Comment:
Since most methods take a variable number of arguments, I'm not sure if this is the best approach. I would just add a simple wrapper for `getSignedUrl` instead.